server/db/db.js

```diff
@@ -144,6 +144,27 @@ const createUsersTable = () =>
     return exist;
   });
 
+const createCallsTable = () =>
+  bookshelf.knex.schema.hasTable('calls').then((exist) => {
+    if (!exist) {
+      return bookshelf.knex.schema.createTable('calls', (table) => {
+        table.increments('id').primary();
+        table.integer('campaign_id').references('campaigns.id').notNullable();
+        table.integer('contact_id').references('contacts.id').notNullable();
+        table.enu('attempt_num', ['1', '2', '3']).defaultTo('1').notNullable();
+        table.unique(['campaign_id', 'contact_id', 'attempt_num']);
+        table.integer('user_id').references('users.id');
+        table.enu('status', ['AVAILABLE', 'ASSIGNED', 'IN_PROGRESS', 'ATTEMPTED']).defaultTo('AVAILABLE').notNullable();
+        table.enu('outcome', ['PENDING', 'ANSWERED', 'BAD_NUMBER', 'DO_NOT_CALL', 'LEFT_MSG', 'NO_ANSWER', 'INCOMPLETE']).defaultTo('PENDING').notNullable();
+        table.string('notes');
+        table.integer('call_sid');
+        table.timestamp('call_started');
+        table.timestamp('call_ended');
```

Comment:
We also should keep the traditional timestamps for Knex (created_at and updated_at).

Reply:
Cool beans, just added.